openapi/components/schemas/key-value-stores/CreateKeyValueStoreResponse.yaml

```
@@ -1,9 +1,9 @@
-title: Getlistofkey-valuestoresresponse
+title: CreateKeyValueStoreResponse
 required:
   - data
 type: object
 properties:
   data:
     allOf:
-      - $ref: ./data11.yaml
+      - $ref: ./KeyValueStore.yaml
```

Btw just learned that if you are not combining multiple definitions together, `allOf` is not needed.

```suggestion
properties:
  data:
      $ref: ./KeyValueStore.yaml
```

No need to fix this, just for the next time.